src/VolleyManagement.WebClient/spa/app/Components/Schedule/schedule.component.ts

```diff
@@ -58,14 +52,16 @@ export class ScheduleComponent implements OnInit {
         if (divisionId === DummyDivisionHeader.DummyHeaderId) {
             return '';
         } else if (divisionId === DummyDivisionHeader.PlayOffHeaderId) {
-            return 'division2';
+            return 'division7';
```

Comment:
probably it is a good idea to move it to some "constants wrapper" class, to be able quickly find/update/refactor it using IDE

Reply:
I thought of that. It's the only place right now. If it will be used somewhere else - we'll introduce one.